diagnostic.md

```diff
@@ -60,19 +63,22 @@ What is the command to generate a new rails-api app?
 What is the command to start an instance of a rails server?
 
 ```bash
-// your response here
+bundle exec rails server or bin/rails server
 ```
 
 What are the commands to drop, create, migrate and seed a database from the command
 line? (5 bullet points)
 
 ```bash
-// your response here
+•bin/rake db:create
+•bin/rake db:migrate
+•bin/rake db:drop
+•bin/rake db:seed
 ```
 
 What is the command to scaffold a pet with a name and age attributes (hint:
 Also think of the data types for each attribute)?
 
 ```bash
-// your response here
+bin/rails generate scaffold post
```

Comment:
Not sure why you said post here, did you mean pet? The command is:

`rails generate scaffold pet name:string age:integer`